modules/understanding-agent-install.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent/installing-with-agent.adoc
+
+:_content-type: CONCEPT
+[id="understanding-agent-install_{context}"]
+= Understanding Agent-based Installer
+As a {product-title} user, you can leverage all the advantages of the Assisted Installer hosted service in disconnected environments.
+
+The Agent installation comprises a bootable ISO that contains the Assisted discovery agent and the Assisted Service; both are required to perform the first cluster installation.
+
+The `openshift-install agent create image` subcommand generates an ephemeral ISO based on the inputs that you provide. You can choose to provide inputs through the following manifests:
+
+Preferred:
+
+* install-config.yaml
```

Comment:
Usually we put `config.yaml`s in backticks. Should these be in backticks? My guess is yes.

Reply:
yes correct